Fix empty output in bash/ip.sh.

icedream opened the pull request:
This pull request is merely a quick fix for the Bash IP read-out script, replacing the `cut` call with `awk`. This allows the script to be more tolerant with indenting spaces.

wzhouwzhou commented:
I am unable to reach the original author of the file, Mohammed Nisham; out of respect, would you mind renaming this as a new file, such as `icedream_ip.sh`?

wzhouwzhou reviewed (changes requested):
(Please see above request about renaming to icedream_ip.sh)

icedream committed:
Add a fixed version of bash/ip.sh based on `awk`.

Original version generated empty output due to `cut` being
whitespace-intolerant.

icedream commented:
Can't find a way to respond to your review anymore but I have done the requested changes now. :)

kannagikazuko committed:
Rename ip_awk.sh to icedream_ip_awk.sh

kannagikazuko committed:
Merge branch 'master' into pr/bash/ip/fix
